dl/src/test/scala/com/intel/analytics/bigdl/optim/OptimizerSpec.scala

```diff
@@ -122,7 +123,28 @@ class OptimizerSpec extends FlatSpec with Matchers {
     loadedModel should be(model)
   }
 
-  it should "save model and state to given path with postfix" in {
+  it should "save model to given hdfs path without postfix" in {
+    val conf: Configuration = new Configuration()
+    val dfs = conf.get("fs.defaultFS")
+    if (dfs != null && dfs.startsWith(File.hdfsPrefix)) {
```

Comment:
if `conf.get("fs.defaultFS")` doesn't return expected `hdfs` prefix, this UT will not actually make sense.
Is there a more friendly way to design this UT?